### PR TITLE
Syntax update of arrays from `array()` to `[]` 

### PR DESCRIPTION
**Pull request by mitogh**
Replace old syntax for arrays on PHP

### CODE REVIEW

**Review by lucatume — CHANGES_REQUESTED** [review]
Look into and fix the tests.

File: src/Tribe/Admin/Help_Page.php [hunk 1]
```diff
@@ -72,50 +72,50 @@ public function register_assets() {
...
+		$plugins['the-events-calendar'] = [
+				'name'        => 'the-events-calendar',
+				'title'       => esc_html__( 'The Events Calendar', 'tribe-common' ),
+				'repo'        => 'https://wordpress.org/plugins/the-events-calendar/',
+				'forum'       => 'https://wordpress.org/support/plugin/the-events-calendar/',
+				'stars_url'   => 'https://wordpress.org/support/plugin/the-events-calendar/reviews/?filter=5',
+				'description' => esc_html__( 'The Events Calendar is a carefully crafted, extensible plugin that lets you easily share your events.', 'tribe-common' ),
+				'is_active'   => false,
+				'version'     => null,
+		];
```
Comment (paulmskim) [inline src/Tribe/Admin/Help_Page.php:86]: this file has weird indentation

File: src/admin-views/tribe-options-help.php [hunk 2]
```diff
@@ -25,13 +25,13 @@
...
-), 10 );
+		sprintf( __( '%s. All of the common (and not-so-common) answers to questions we see are here. It’s often the fastest path to finding an answer!', 'tribe-common' ), '<strong><a href="http://m.tri.be/18j9" target="_blank">' . __( 'Check our Knowledgebase', 'tribe-common' ) . '</a></strong>' ),
+		sprintf( __( '%s. Testing for an existing conflict is the best start for in-depth troubleshooting. We will often ask you to follow these steps when opening a new thread, so doing this ahead of time will be super helpful.', 'tribe-common' ), '<strong><a href="http://m.tri.be/18jh" target="_blank">' . __( 'Test for a theme or plugin conflict', 'tribe-common' ) . '</a></strong>' ),
+		sprintf( __( '%s. There are very few issues we haven’t seen and it’s likely another user has already asked your question and gotten an answer from our support staff. While posting to the help desk is open only to paid customers, they are open for anyone to search and review.', 'tribe-common' ), '<strong><a href="http://m.tri.be/4w/" target="_blank">' . __( 'Search our support help desk', 'tribe-common' ) . '</a></strong>' ),
+], 10 );
```
Comment (paulmskim) [inline src/admin-views/tribe-options-help.php:34]: indentation

**Review by paulmskim — COMMENTED** [review]
a couple of odd indentations

**Review by bordoni — APPROVED** [review]

**Review by bordoni — CHANGES_REQUESTED** [review]
We need to make sure the syntax problems are resolved. Check tests.

**Comment by lucatume** [comment]
Rebasing on `release/B20.12`.

**Pull request merged**